.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,37 @@
+pr:
+  branches:
+    include:
+    - master
+    - search-preview
+    - BuildToolsForSdk
+
+variables:
+  DotNetCoreVersion: '2.1.503'
+
+jobs:
+  - job: Windows_Build
+    pool:
+      vmImage: 'VS2017-Win2016'
+    variables:
+      IgnoreBuildTokens: ''
```

Comment:
If you don't use this you can just remove it here and in the msbuildARguments below.